pkg/resolution/resolver/cluster/resolver_test.go

```diff
@@ -110,10 +110,11 @@ func TestValidateParamsNotEnabled(t *testing.T) {
 
 func TestValidateParamsFailure(t *testing.T) {
 	testCases := []struct {
-		name        string
-		params      map[string]string
-		conf        map[string]string
-		expectedErr string
+		name          string
+		params        map[string]string
+		conf          map[string]string
+		expectedErr   string
+		expectedNoErr bool
```

Comment:
Can an empty string be used to indicate that we expect no error, instead of introducing `expectedNoErr`

Reply:
sounds good! thanks for the suggestion